src/node_errors.cc

```diff
@@ -978,6 +978,7 @@ void TriggerUncaughtException(Isolate* isolate,
 
   // Now we are certain that the exception is fatal.
   ReportFatalException(env, error, message, EnhanceFatalException::kEnhance);
+  WaitForInspectorDisconnect(env);
```

Comment:
I didn't leave this for other `Exit()` calls in `TriggerUncaughtException()` as those cases are more abnormal than this and I think it would unsafe to wait until the inspector to shut down in those cases.